tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -333,7 +333,10 @@
               ],
               "validation_context": {
                 "trusted_ca": {
-                  {{- if eq .pilot_cert_provider "kubernetes" }}
+                  {{- if .provided_cert }}
+                  {{ $combine := (printf "%s%s" .provided_cert "/root-cert.pem")}}
```

Comment:
nit: don't make a variable here. Can you just do `"filename": {{(printf "%s%s" .provided_cert "/root-cert.pem") }}`

Reply:
fixed